src/components/dataset-query/dataset-query.js

```diff
@@ -37,16 +43,41 @@ const DatasetQuery = props => {
   const updateLocalQueryText = e => setLocalQueryText(e.target.value)
   const updateReduxQueryText = (e) => setQueryText(e.target.value)
 
-  const textArea = <textarea rows={5} type='text' value={localQueryText} onBlur={updateReduxQueryText} onChange={updateLocalQueryText} className='query-input' />
+  const textArea = <textarea rows={5} type='text' value={localQueryText} onBlur={updateReduxQueryText} onChange={updateLocalQueryText} className='sql-textbox' />
   const submitButton = <button className="action-button" disabled={isQueryLoading} onClick={submit}>Submit</button>
   const cancelButton = <button className="action-button" disabled={!isQueryLoading} onClick={cancel}>Cancel</button>
-  
+
   const showSuccessMessage = !queryFailureMessage && userHasInteracted && !isQueryLoading
   const successMessage = showSuccessMessage && (
     <span className='success-message'>
       Query successful.  To refresh the visualization, you must change an element in the trace
     </span>
   )
+  const createHoverText = (systemName) => showTooltipCopied ? 'Copied!' : `Copy table name '${systemName}'`
+
+  const recommendationItems = () => {
+    return recommendations.map(rec => {
+      const tooltipId = `tool-tip-${rec.id}`
+      return (
+        <div key={rec.id}>
+          <a className="recommended-dataset" href={RecommendationUtils.getDatasetUrl(rec)} target='_blank'>
+            {rec.dataTitle}
+          </a>
+          <ReactTooltip id={tooltipId} place="right" effect="solid" afterHide={() => setShowTooltipCopied(false)} getContent={() => createHoverText(rec.systemName)} />
+          <CopyToClipboard text={rec.systemName} onCopy={() => setShowTooltipCopied(true)}>
+            <AssignmentOutlinedIcon data-for={tooltipId} className="copy-table-name-icon" data-tip />
+          </CopyToClipboard>
+        </div>)
+    })
+  }
+
+  const recommendationList = () => {
+    return (
+      <div className="recommendation-list">
+        {recommendationItems()}
+      </div>
+    )
+  }
```

Comment:
Would it be possible to break this list into its own component?  It could be connected to the recommendations in redux - then the `dataset-query` wouldn't need to know anything about recommendations.

Reply:
Yeah, we sorta crossed the rubicon on that being part of another component didn't we

Reply:
We're putting it into a new component but I _think_ the team has normed on only having connectors on "page" components and then passing the data down from there.